service/src/main/java/bio/terra/workspace/service/resource/controlled/cloud/gcp/ainotebook/ControlledAiNotebookInstanceResource.java

```diff
@@ -261,8 +261,8 @@ public void validate() {
   }
 
   /** Returns an auto generated instance name with the username and date time. */
-  public static String generateInstanceId(@Nullable String email) {
-    String mangledUsername = mangleUsername(extractUsername(email));
+  public static String generateInstanceId(@Nullable String instanceName) {
```

Comment:
i'm confused why we still have this. Now there are two ways to create instance name. In your PR description, you said for ai notebook, we are leaving the name as is. But here we are attaching date.

Reply:
based on the code, it generate when the cloudname is null. 

can you delete them and use getHandler().generateCloudName instead to be consistent?

Reply:
Done. I use the generateCloudName to make it consistent. Please help me review it.